docs/GettingStarted.md

```diff
@@ -72,6 +72,7 @@ of the Datadog tracer, you can override the following defaults:
       default_controller_service: 'rails-controller',
       default_cache_service: 'rails-cache',
       default_database_service: 'postgresql',
+      distributed_tracing_enabled: false,
```

Comment:
we missed that in the doc; mostly the goal is to remember that with the current config version, you can enable distributed tracing from the settings. Not that great since we're planning to change that, but at least we have a reference for that.